opencontrail/files/2.2/config.global.js

```diff
@@ -1,4 +1,4 @@
-{%- from "opencontrail/map.jinja" import web with context %}
+{%- from "opencontrail/map.jinja" import common,web with context %}
```

Comment:
Readme has info of supported version 3.0.x. Do you really want to remove any change under files/2.2?